permissions/plugin-pipeline-milestone-step.yml

```diff
@@ -4,4 +4,9 @@ paths:
 - "org/jenkins-ci/plugins/pipeline-milestone-step"
 developers:
 - "amuniz"
+- "svanoort"
+- "abayer"
 - "jglick"
+- "rsandell"
+- "stephenc"
```

Comment:
:-1: 

Beware, it's the Jenkins LDAP/JIRA usernames here, not GitHub.

Use https://issues.jenkins-ci.org/secure/ViewProfile.jspa?name=stephenc URL to check.

@stephenc is https://issues.jenkins-ci.org/secure/ViewProfile.jspa?name=stephenconnolly